readme.md

```diff
@@ -429,6 +429,15 @@ export default ({ href, name }) =>
   </Link>
 ```
 
+##### Disabling the scroll changes to top on page
+
+The default behaviour of scrolling for the `<Link>` component is to scroll to the top of the page except hashtag. When using `<Link>` to update the same page with query params without scrolling, you can use `scroll={false}` props to prevent the scroll changes.
```

Comment:
The default behaviour of the `<Link>` component is to scroll to the top of the page. When there is a hash defined it will scroll to the specific id, just like a normal `<a>` tag. 
To prevent scrolling to the top / hash `scroll={false}` can be added to `<Link>`: